Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:6 AS builder
+
+ENV ENV_NODE=production
+
+RUN groupadd --gid 1100 lisk && \
+    useradd --create-home --home-dir /home/lisk --shell /bin/bash --uid 1100 --gid 1100 lisk
+# As of May 2018 cloud.docker.com runs docker 17.06.1-ce
+# however version 17.12 is required to use the chown flag
+COPY . /home/lisk/lisk/
```

Comment:
We shouldn't just copy every thing to docker. First run `grunt release` and copy the compiled source code. You may not find any noticeable difference, but any future change in Gruntfile may break the docker. So better to use compile release after grunt.

Reply:
This is the  build container, we must copy everything in order to be able to run `npm install` or `grunt release`. Regarding the latter I was told to remove it; @diego-G can you please confirm once more?

Reply:
@fchavant @diego-G I am sure we need to run `grunt release` and copy only the archive files to container. 

@fchavant We can run `npm install` and `grunt release` on host machine, where we are actually building container. I think we don't need a seperate container just to run `grunt release`.

Reply:
> I think we don't need a seperate container just to run grunt release.
It is a way to have a clean build environment w/o polluting the final containers with build time dependencies.

> I am sure we need to run grunt release and copy only the archive files to container.
What would be the advantage in using the tarball `grunt` creates? It's meant to be consumed by lisk-build's `build.sh` script which makes sense for all-in-one release but not docker.

Reply:
@fchavant Nothing will be polluted, if you run `grunt release` on host machine and copy the archive build file to container. It will not contain `node_modules` directory. So you will do a clean `npm install` in container.